app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/SessionDetailsFragment.kt

```diff
@@ -112,113 +120,153 @@ class SessionDetailsFragment : Fragment(), SessionDetailsViewModel.ViewActionHan
+                textView.applyMarkdown(typeface, model.description)
             }
+        }
 
-            // Session online
-            val sessionOnlineSectionView = view.requireViewByIdCompat<TextView>(R.id.session_details_content_session_online_section_view)
-            typeface = typefaceFactory.getTypeface(viewModel.sessionOnlineSectionFont)
-            sessionOnlineSectionView.typeface = typeface
-            val sessionOnlineLinkView = view.requireViewByIdCompat<TextView>(R.id.session_details_content_session_online_view)
-            if (viewModel.hasWikiLinks) {
+        // Links
+        val linksView = view.requireViewByIdCompat<TextView>(R.id.session_details_content_links_section_view)
+        textView = view.requireViewByIdCompat(R.id.session_details_content_links_view)
+        if (model.hasLinks) {
+            typeface = typefaceFactory.getTypeface(viewModel.linksSectionFont)
+            linksView.typeface = typeface
+            MyApp.LogDebug(LOG_TAG, "show links")
+            linksView.isVisible = true
+            typeface = typefaceFactory.getTypeface(viewModel.linksFont)
+            textView.applyHtml(typeface, model.formattedLinks)
+        } else {
+            linksView.isVisible = false
+            textView.isVisible = false
+        }
+
+        // Session online
+        val sessionOnlineSectionView = view.requireViewByIdCompat<TextView>(R.id.session_details_content_session_online_section_view)
+        typeface = typefaceFactory.getTypeface(viewModel.sessionOnlineSectionFont)
+        sessionOnlineSectionView.typeface = typeface
+        val sessionOnlineLinkView = view.requireViewByIdCompat<TextView>(R.id.session_details_content_session_online_view)
+        if (model.hasWikiLinks) {
+            sessionOnlineSectionView.isVisible = false
+            sessionOnlineLinkView.isVisible = false
+        } else {
+            val sessionLink = model.sessionLink
+            if (sessionLink.isEmpty()) {
                 sessionOnlineSectionView.isVisible = false
                 sessionOnlineLinkView.isVisible = false
             } else {
-                val sessionLink = viewModel.sessionLink
-                if (sessionLink.isEmpty()) {
-                    sessionOnlineSectionView.isVisible = false
-                    sessionOnlineLinkView.isVisible = false
-                } else {
-                    sessionOnlineSectionView.isVisible = true
-                    sessionOnlineLinkView.isVisible = true
-                    typeface = typefaceFactory.getTypeface(viewModel.sessionOnlineFont)
-                    sessionOnlineLinkView.applyHtml(typeface, sessionLink)
-                }
+                sessionOnlineSectionView.isVisible = true
+                sessionOnlineLinkView.isVisible = true
+                typeface = typefaceFactory.getTypeface(viewModel.sessionOnlineFont)
+                sessionOnlineLinkView.applyHtml(typeface, sessionLink)
             }
```

Comment:
You could fuse those branches or is there a reason not to do it?:
```
val sessionLink = model.sessionLink
        
if (model.hasWikiLinks || sessionLink.isEmpty()) {
    sessionOnlineSectionView.isVisible = false
    sessionOnlineLinkView.isVisible = false
} else {
    sessionOnlineSectionView.isVisible = true
    sessionOnlineLinkView.isVisible = true
    typeface = typefaceFactory.getTypeface(viewModel.sessionOnlineFont)
    sessionOnlineLinkView.applyHtml(typeface, sessionLink)
}
```

Reply:
:heavy_check_mark: Applied in 705739fa159e2ce83615bf013a493b145b39d48a.